### ISSUE TITLE
My repo is marked as (inaccessible) even if I created the "correct" token

### ISSUE THREAD

**Issue by andreagrandi** [ptsochantaris/trailer#489]
Hi, I created the GitHub token using the requested permissions in the first dialog (which are also the same mentioned here https://github.com/ptsochantaris/trailer/issues/432#issuecomment-846402660 ) and I've added the repository I'm interested in. But there is a problem: the repository appears as "(inaccessible)" in the list and no notifications are coming from that repo. How do I fix this? Thanks

**Comment by ptsochantaris** [comment]
Hi, and sorry for the very late reply - can you check the API log from the "Misc" tab in preferences to see what kind of error you get from the API when trying to access that repo info?

**Comment by andreagrandi** [comment]
> Hi, and sorry for the very late reply - can you check the API log from the "Misc" tab in preferences to see what kind of error you get from the API when trying to access that repo info?

Hi, no worries about the delay and thanks for your reply 🙂

First of all, to have a cleaner logs, I marked as "Hide" all the repositories except the one I'm interested in. Once I've done that, I clicked on the "Refresh" menu option and I got this in the logs:

```logs
>>> 20231018 13:01:03:352 - Actively verifying reported network availability state…
>>> 20231018 13:01:03:352 - No change to network state
>>> 20231018 13:01:03:354 - Starting refresh
>>> 20231018 13:01:03:355 - Status update: Fetching…
>>> 20231018 13:01:03:358 - Will sync items from: 
>>> 20231018 13:01:03:369 - Updating general PullRequest menu, 0 total items
>>> 20231018 13:01:03:408 - Status update: Processing update…
>>> 20231018 13:01:03:408 - Nuked total 0 items marked for deletion
>>> 20231018 13:01:03:409 - Committing synced data
>>> 20231018 13:01:03:409 - Synced data committed
>>> 20231018 13:01:03:409 - No DB changes
>>> 20231018 13:01:03:410 - Postprocess done - 0.0010039806365966797 sec
>>> 20231018 13:01:03:413 - No DB changes
>>> 20231018 13:01:03:413 - Refresh done
>>> 20231018 13:01:03:413 - Status update: Updated just now
>>> 20231018 13:01:03:415 - (Re)creating settings cache
>>> 20231018 13:01:03:418 - Updating general PullRequest menu, 0 total items
```
This is what I see in "Repositories" settings (oh, by the way... if there was a way to completely remove all the other entries, that would be great. I had to manually Hide all of them, one bye one 😓)

![Screenshot 2023-10-18 at 13 04 15](https://github.com/ptsochantaris/trailer/assets/636391/2c369c77-e7f1-4b22-b930-547defef15ef)

Another information I can give you is that the "inaccessible" repository is part of an organisation and that the repository is private. Does it need specific/additional permissions (other than those I specified as in my first comment here) ?

Thanks

**Comment by andreagrandi** [comment]
p.s: meanwhile I can paste you this from the logs: `20231018 14:41:28:901 - (GitHub GET https://api.github.com/repos/MY_ORG/MY_REPO - RESULT: Success`

**Comment by ptsochantaris** [comment]
Yes please, don't post logs with private info here - But if you are OK with me seeing them please feel free to email them over to ptsochantaris at me dot com and I'll happily take a look.

**Comment by javierlopezdeancos** [comment]
How this is going? I have the same issue

**Comment by ptsochantaris** [comment]
@javierlopezdeancos Hi, do you have any logs related to this issue? Is there perhaps some way I can try to replicate this?

**Comment by othonreyes** [comment]
> How this is going? I have the same issue

I had the same issue. What I ended doing was regenerating the token, using API v4 and syncing again. worked like a charm!

(i don't have logs, no idea where to get them)

**Comment by ptsochantaris** [comment]
@othonreyes thanks a lot for sharing this, it may help others who run into the issue. 

I strongly suspect GitHub at some point modified access permissions which required tokens to be re-created to maintain access to various things... but would love to know more about the root cause. 

If you do run into problems, you can always go to Misc->Stream Log and copy all or parts of your log that you think are related to the issue, it can definitely help debug or replicate stuff.

**Comment by StianFlatby** [comment]
I had this issue as well. There were no obvious errors in the log stream after "Reload all data". The repo just said "Inaccessible".

I enabled v4, which did not work. Then I regenerated the token and did "Reload All Data", that fixed it 👌 
Sorry for not keeping a version of the logs from when it didn't work 😅

**Comment by ptsochantaris** [comment]
It's been a while, but v1.9.0 is now up and has a lot of fixes and improvements to sync logic. If you feel like giving it a go to see if it improves things for you, you can find it here: https://github.com/ptsochantaris/trailer/releases/tag/v1.9.0
